scanapi/tree/request_node.py

```diff
@@ -98,7 +100,11 @@ def body(self):
 
     @property
     def tests(self):
-        return (TestingNode(spec, self) for spec in self.spec.get("tests", []))
+        return (TestingNode(spec, self) for spec in self.spec.get(TESTS_KEY, []))
+
+    @property
+    def retry(self):
```

Comment:
Does anyone think I should add something like `RetryNode`? I opted not to do it because if one decide to expose all the possible options of Retry, that would pollute the proper keys.

Reply:
I believe it is ok not having the node here. I agree with you 👍